src/tz_linux.rs

```diff
@@ -89,7 +89,7 @@ mod openwrt {
             }
         }
 
-        timezone.ok_or_else(|| crate::GetTimezoneError::OsError)
+        timezone.ok_or(crate::GetTimezoneError::OsError)
```

Comment:
`crate::GetTimezoneError::OsError` is `Copy` which is why clippy says the closure is not needed.